conda/base/context.py

```diff
@@ -444,6 +444,14 @@ def experimental_solver(self):
         aliases=("conda-build", "conda_build"),
     )
 
+    ####################################################
+    #               Plugin Configuration               #
+    ####################################################
+
+    no_plugins = ParameterLoader(
+        PrimitiveParameter(NO_PLUGINS, element_type=(bool, NoneType)),
+    )
```

Comment:
Lets simplify this to only allow True/False:

```suggestion
    no_plugins = ParameterLoader(PrimitiveParameter(NO_PLUGINS))
```